hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/SubsystemExtension.java

```diff
@@ -58,7 +58,7 @@ public void initialize(ExtensionContext context) {
                 MAJOR_VERSION, MINOR_VERSION, MICRO_VERSION);
 
         // This subsystem should be runnable on a host
-        subsystem.setHostCapable();
+        //subsystem.setHostCapable();
```

Comment:
Mark it as TODO and continue
We need to 1st get this to work in standalone mode before we need to look at domain mode.
Also I believe that a EAP7 domain controller can control EAP6 managed servers, so that we could shoot for a 'hybrid' if all fails.